infrastructure/build.gradle.kts

```diff
@@ -9,6 +9,15 @@ tasks.register("setUpRabbitMQ") {
     finalizedBy("executeScript")
 }
 
+tasks.register("setUpFireMock") {
+    project.ext["args"] = ""
+    project.ext["linuxScript"] = "./SetUpFireMock.sh"
+    project.ext["windowsScript"] = ""
```

Comment:
You could include the Windows equivalent if you don't mind. I think @awaisathar could help you test it.